### PR TITLE
Makefile: add "flash" target

### PR DESCRIPTION
**Pull request by tjurtsch**

### CODE REVIEW

**Review by tmichalak — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,18 @@ else
...
 
+flash: FORCE
+ifeq ($(TARGET),zcu104)
+	@echo "For ZCU104 please copy the file build/zcu104/gateware/zcu104.bit to the boot partition on microSD card"
+	@exit 1
+else
+	python rowhammer_tester/targets/$(TARGET).py --flash $(TARGET_ARGS)
+ifeq ($(TARGET),lpddr4_test_board)
+	# Enable Quad mode in spi flash module
+	openocd -f prog/openocd_xc7_ft4232.cfg -c "init; jtagspi_init 0 prog/bscan_spi_xc7k70t.bit; jtagspi write_cmd 1 512 16 0; exit"
```
[the remark this answers is not in the document]
Comment (tmichalak) [reply Makefile:57]: I agree, this is something that has to be set only once on a fresh Flash module, provided it's not being cleared by a Xilinx Platform Cable. It also takes negligible amount of time.
However, we could just override the OpenOCD class and its flash method, this shouldn't require too much effor.

**Comment by tjurtsch** [comment]
A minor bug was introduced during cleanup. QUAD bit isn't set correctly during first call. Second call fixes that. I'm looking into the issue.

**Comment by tjurtsch** [comment]
I resolved the mentioned issue with double write in openocd. We need to wait for Write-In-Progress bit in status register to reset after a call to WRR (write configuration register) command. https://www.cypress.com/file/448601/download

**Review by tmichalak — APPROVED** [review]
LGTM

**Comment by tjurtsch** [comment]
The patch for OpenOCD was submitted for a review http://openocd.zylin.com/#/c/6208/1

**Pull request merged**